components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/KeyManagersApiServiceImpl.java

```diff
@@ -25,20 +25,15 @@
 
     public Response keyManagersGet(String xWSO2Tenant, MessageContext messageContext) {
 
-        String tenantDomain = RestApiCommonUtil.getLoggedInUserTenantDomain();
-
-        if (StringUtils.isNotEmpty(xWSO2Tenant)) {
-            tenantDomain = xWSO2Tenant;
-        }
-
+        String organization = RestApiUtil.getOrganization(messageContext);
         APIAdmin apiAdmin = new APIAdminImpl();
         try {
             List<KeyManagerConfigurationDTO> keyManagerConfigurations =
-                    apiAdmin.getKeyManagerConfigurationsByTenant(tenantDomain);
+                    apiAdmin.getKeyManagerConfigurationsByTenant(organization);
```

Comment:
Shall we rename this method?
```suggestion
                    apiAdmin.getKeyManagerConfigurationsByOrganization(organization);
```

Reply:
Fixed via 7d86bcf96daa8bbd55e9b701d7d49b6225cbbb75